src/accountLinks.js

```diff
@@ -0,0 +1,78 @@
+const { createLink } = require('3id-blockchain-utils')
+const utils = require('./utils')
+
+class AccountLinks {
+  constructor (ceramic, provider) {
+    this._ceramic = ceramic
+    this.provider = provider
+  }
+
+  async create (address, did, proof = null) {
+    const doc = await this._ceramic.createDocument(null, 'account-link', {
+      owners: [await this._convertToCaip10(address)]
+    })
+    if (!proof) {
+      if (!this.provider) {
+        throw new Error('Provider must be set to create an account link')
+      }
+      proof = await createLink(did, address, this.provider)
+    }
+    await doc.change(proof)
+    return doc.content
+  }
+
+  async read (address) {
+    const docId = await this._getDocId(address)
+    const doc = await this._ceramic.loadDocument(docId, { onlyGenesis: false })
+    return doc.content
```

Comment:
Instead of getting docId from the first method call here you should just return the document. We don't need to load the document if we already created it.